lte/gateway/c/oai/tasks/grpc_service/AmfServiceImpl.cpp

```diff
@@ -43,8 +43,94 @@ Status AmfServiceImpl::SetAmfNotification(
 Status AmfServiceImpl::SetSmfSessionContext(
     ServerContext* context, const SetSMSessionContextAccess* request,
     SmContextVoid* response) {
-  OAILOG_INFO(LOG_UTIL, "Received  GRPC SetSMSessionContextAccess request\n");
-  // ToDo processing ITTI,ZMQ
+  OAILOG_INFO(LOG_UTIL, "Received  GRPC SetSMSessionContextAccess request \n");
+  OAILOG_INFO(
+      LOG_UTIL, "Details : Flow iden %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .qos_list()
+          .flow()
+          .qos_flow_ident());
+  OAILOG_INFO(
+      LOG_UTIL, " FiveQi %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .qos_list()
+          .flow()
+          .param()
+          .qos_chars()
+          .fiveqi());
+  OAILOG_INFO(
+      LOG_UTIL, " prio %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .qos_list()
+          .flow()
+          .param()
+          .alloc_reten_prio()
+          .prio_level());
+  OAILOG_INFO(
+      LOG_UTIL, " prem_cap %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .qos_list()
+          .flow()
+          .param()
+          .alloc_reten_prio()
+          .pre_emtion_cap());
+  OAILOG_INFO(
+      LOG_UTIL, " prem_vuner%d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .qos_list()
+          .flow()
+          .param()
+          .alloc_reten_prio()
+          .pre_emtion_vul());
+
+  OAILOG_INFO(
+      LOG_UTIL, "  Session Ambr down link type %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .session_ambr()
+          .downlink_unit_type());
+  OAILOG_INFO(
+      LOG_UTIL, "  Session Ambr up link type %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .session_ambr()
+          .uplink_unit_type());
+  OAILOG_INFO(
+      LOG_UTIL, "  Session Ambr down link units %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .session_ambr()
+          .downlink_units());
+  OAILOG_INFO(
+      LOG_UTIL, "  Session Ambr up link type %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .session_ambr()
+          .uplink_units());
+  OAILOG_INFO(
+      LOG_UTIL, "  procedure tran iden  %d \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .procedure_trans_identity());
+  OAILOG_INFO(
+      LOG_UTIL, "   upf ip address %s \n",
+      request->rat_specific_context()
+          .m5g_session_context_rsp()
+          .upf_endpoint()
+          .end_ipv4_addr()
+          .c_str());
+  OAILOG_INFO(
+      LOG_UTIL, "   upf teid %u \n",
+      stoi(request->rat_specific_context()
+               .m5g_session_context_rsp()
+               .upf_endpoint()
+               .teid()));
+
```

Comment:
Can you make this debug level and use single OAILOG_DEBUG call (too many lines, put them in a single line as well).

Reply:
This is not neded , I removed all of them for now. Anyway AMF team is going re-write all these routines.